lib/tailwind/src/components/templates/numberTemplate.tsx

```diff
@@ -0,0 +1,40 @@
+import z from "zod";
+import { useErrorsAtPath, useFormDataAtPath } from "../..";
+import { ErrorsListTemplate } from "./errorsListTemplate";
+import React from "react";
```

Comment:
why we need this import

Reply:
TS2686: React refers to a UMD global, but the current file is a module. Consider adding an import instead.